releases/latest/mysql-k8s-bundle.yaml

```diff
@@ -21,14 +16,19 @@ applications:
     channel: latest/edge
     charm: tls-certificates-operator
     constraints: arch=amd64
-    revision: 15
-    scale: 1
```

Comment:
No more pinning for tls-certificates-operator?

Reply:
odd. we should be pinning the tls-certificates-operator. will investigate the script that generates the revisions and trace the reason why the pinning is skipped

Reply:
Ignore. I was blind. It was resorted (see line 22-23).